vspace/src/main/java/edu/asu/diging/vspace/web/staff/SpaceController.java

```diff
@@ -35,23 +37,27 @@
 
     @RequestMapping("/staff/space/{id}")
     public String showSpace(@PathVariable String id, Model model) {
-    	
-    	
+
+
         ISpace space = spaceManager.getFullyLoadedSpace(id);
         model.addAttribute("space", space);
         model.addAttribute("externalLinks", linkManager.getExternalLinkDisplays(id));
-        model.addAttribute("spaceLinks", linkManager.getSpaceLinkDisplays(id));
+        if(space.getShowUnpublishedLinks()!=null && space.getShowUnpublishedLinks().equals(ShowUnpublishedLinks.YES)) {
+            model.addAttribute("spaceLinks",linkManager.getSpaceLinkDisplays(id));
+        }else {
+            model.addAttribute("spaceLinks", linkManager.getSpaceLinkDisplaysForPublishedSpace(id, SpaceStatus.PUBLISHED));
+        }
```

Comment:
this if for the staff view, right? It should always show all space links (otherwise you can't edit them).